reference/service/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Pulumi Cloud Console
+---
+
+[The Pulumi Cloud Console](https://app.pulumi.com) allows you to manage your stacks online. It enables
+you to to collaborate with multiple developers, protect against concurrent updates, store resource
+history, integrate with your CI/CD system, and more!
+
+## Getting Started
+
+To get started using the Pulumi Cloud Console, go to [app.pulumi.com](https://app.pulumi.com) and
+sign in with your GitHub account.
+
+> **NOTE**
+>
+> Support for more kinds of identity providers is coming soon. If you require something
```

Comment:
I would say something like "Support for more kinds of identity providers is coming soon to the Team Edition, and available for privately hosted Enterprise Edition instances. If you require ..."

Reply:
Done.